site/docs/plugins/conversations.md

```diff
@@ -923,14 +927,14 @@ Basically, the bot loses its train of thought, and the user has to start over.
 This is bad design and annoying.
 
 **Blocking.**
-If wait calls would block until the next update arrives, it means that the middleware execution for the first update can't complete until the entire conversation completes.
+If `wait` calls would block until the next update arrives, it means that the middleware execution for the first update can't complete until the entire conversation completes.
```

Comment:
By _wait calls_, I mean calls to any of the wait* methods. Saying `wait` calls would imply only `wait`, which is inaccurate. Please revert this and the following similar changes.